PP_APP/client/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
```

Comment:
did you use bootstrap to automatically generate the html/css? or was it django generated?

Reply:
So anything in the `client` folder is not related to Django. The 'index.html' file is created manually. All HTML files have this boilerplate code: https://www.sitepoint.com/a-basic-html5-template/ where the content on interest will go inside the <body> tags.